config/exports/apiresourceschema-edgeplacements.edge.kcp.io.yaml

```diff
@@ -16,18 +16,23 @@ spec:
   versions:
   - name: v1alpha1
     schema:
-      description: "EdgePlacement binds a collection of (a) Namespaces and non-namespaced
-        objects to (b) a collection of Locations. An EdgePlacement object appears
-        in the center and directs all the syncable objects in the selected Namespaces
-        and all the selected non-namespaced objects to propagate to _all_ of the selected
-        Locations (one SyncTarget in each such Location). This is not entirely unrelated
-        to a TMC Placement, which directs the selected Namespaces to propagate to
-        _one_ of the selected Locations. \n Overlap between EdgePlacements is allowed:
-        two different EdgePlacement objects may select intersecting Location sets
-        and/or intersecting Namespace sets. This is not problematic because: - propagation
-        _into_ a destination is additive; - propagation _from_ a source is additive;
-        - two directives to propagate the same object to the same destination are
-        simply redundant (they, by design, can not conflict)."
+      description: "EdgePlacement exists in the center and binds (a) a collection
+        of Locations with (b) both (b1) objects in the center to downsync (propagate
+        desired state from center to edge and return reported state from edge toward
+        center), and (b2) a way identifying objects (in edge clusters) to upsync (propagate
```

Comment:
should "a way identifying" be "a way of identifying"?

Reply:
Yes, thanks.

Reply:
Fixed in the force-push to a197cae